test/hotspot/jtreg/vmTestbase/nsk/share/Log.java

```diff
@@ -223,21 +220,12 @@ public void enableVerbose(boolean enable) {
         if (!verbose) {
             flushLogBuffer();
         }
-        verbose = enable;
```

Comment:
I assume this API should never be called with "false" as an argument. Maybe we should throw an exception if it is.

Reply:
Done.